meta/documents/user_guide/en/obtaining_credentials.md

```diff
@@ -0,0 +1,44 @@
+# Wayfair plugin: Getting API Credentials
+
+In order for the Wayfair plugin to connect to Wayfair's systems, you need to provide your unique API credentials.
+
+To receive the API credentials for you organization, follow these steps:
+
+## 1. Send an Email to Wayfair
+
+Send an email to ERPSupport@wayfair.com to request assistance:
+
+- Subject : Access to Plentymarkets plugin / "Name of your company" (SuID)
+- Body:
+    - Contact information
+    - Descriptions of your organizations' plans and needs
+    - Any important dates regarding Plentymarkets >> Wayfair integrations.
+
+You will promptly receive a response containing these details:
+- Confirmation of access to
+- Supplier ID(s)
+
+## 2. Generate Application Credentials
+
+1. Log in to your Partner Home account at partners.wayfair.com
+
+2. Find the `Developer` menu. It may be on the top banner of the page. Otherwise, it is accessible in the `More` menu in the banner.
+
+3. Activate the `Developer` menu, and click on `Application`. You should be redirected to a new page.

```

Comment:
We can simplify these three steps by telling them to go to https://partners.wayfair.com/developer/applications

Not sure if it's that simple because it doesn't prompt me to log in but if we can use direct links that would make things easier.

Reply:
Yeah I was tempted to switch this old text to tell them to use the direct link but also didn't know if everything would be cool auth-wise.

Reply:
It seems to prompt for auth when I used an incognito window.
@dFed17 I know you were discussing this with the API guys. Any input?

Reply:
So you'll need to login to partners home, but the redirect post-login should resolve to the clicked link. I'd also opt for the URL, as it should be accessible to ALL partners home users. 

The developer user role adds the `developer` tab in a user's nav bar, but doesn't actually affect access to the page.

Reply:
Yeah so you'll need to log into partners home if the user doesn't have an active session - though it once they enter their partners home creds, the browser should continue to the clicked link. 

URL is probably a good move in my opinion, as users wont be blocked by not having the developer role. The developer role adds the developer tab to a users nav bar, but the pages are accessible for everyone. 

Best might be to describe how to get there via the Nav bar (as you're doing), and maybe add an `OR visit the following URL` type of thing. My hesitation with only using the URL is that users won't actual know how to find the tool without finding the link

Reply:
I'm going with the URL. They can bookmark it. If the UX is not intuitive, the Partner Home team should help.